lib/layouts.js

```diff
@@ -92,7 +94,7 @@ function colorize(str, style) {
 function timestampLevelAndCategory(loggingEvent, colour, timezoneOffset) {
   return colorize(
     formatLogData(
-      '[%s] [%s] %s - '
+      (process.env.LOG4JS_USE_STACKTRACE ? '[%s] [%s] %s %s %s - ' : '[%s] [%s] %s - ')
```

Comment:
I can see you've changed the format, but I don't see you adding the `loggingEvent.lineinfo`or loggingEvent.fileinfo` values to the format arg list? Looks like you've got two extra `%s` values but nothing filling them?